src/addAndRemove.js

```diff
@@ -0,0 +1,56 @@
+export const addToDo = (input, todoListData) => {
+  const dataObj = {
+    index: todoListData.length,
+    description: '',
+    completed: false,
+  };
+
+  dataObj.description = input;
+  todoListData.push(dataObj);
+};
+
+export const modifyList = (todoListData, saveToLocalStorage, createIndexes, refreshPage) => {
```

Comment:
- [ ] **As per the project requirements, you have to create three different functions to `update`, `clear`, and `remove` tasks. Please refactor this function into three different functions.**

Why? 

Firstly, currently, your function has more than 50 lines. It contains a lot of code and is performing three different tasks. It is difficult to understand and maintain the function.

Secondly, in future modules, you will learn a principle called Single Responsibility Principle. It states that a `Class` or a `Function` should only be responsible for one thing. They should only perform one task. This helps us to avoid code smell and write understandable and maintainable code.

Reply:
This issue is well resolved! ✔️